exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillCursor.java

```diff
@@ -66,17 +69,18 @@
   private boolean afterFirstBatch = false;
 
   /**
-   * Whether the next call to this.next() should just return {@code true} rather
-   * than calling nextRowInternally() to try to advance to the next
-   * record.
+   * Whether the next call to {@code this.}{@link #next()} should just return
+   * {@code true} rather than calling {@link #nextRowInternally()} to try to
+   * advance to the next record.
    * <p>
-   *   Currently, can be true only for first call to next().
+   *   Currently, can be true only for first call to {@link #next()}.
    * </p>
    * <p>
-   *   (Relates to loadInitialSchema()'s calling nextRowInternally()
+   *   (Relates to {@link #loadInitialSchema()}'s calling nextRowInternally()
```

Comment:
Frequently, for second and later references in the same paragraph or adjacent paragraphs, I avoid making those later references links, since having more links can make the text less readable.

However, since here I already have multiple links to next(), I did make that nextRowInternally() a link (and added a missing {@code...} for "Statement.execute...(...)").